ros_google_cloud_language/node_scripts/analyze_text.py

```diff
@@ -8,6 +8,8 @@
 import actionlib
 import rospy
 import os
+import sys
+from unidecode import unidecode
```

Comment:
This import failes in `python2`, so please add `try`, `except`

Reply:
Thank you. Updated. [cc249d2](https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/488/commits/cc249d29e611fc563119efbab6a1905e5a2307e8)